.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,259 @@
+      - name: Build plugin
+        run: |
+          ./mvnw --errors --batch-mode --show-version ${{ inputs.maven_args }} ${{ inputs.maven_goal }}
+      - name: Install plugin
+        run: |
+          kpm install_java_plugin ${{ inputs.plugin_name }} --from-source-file target/${{ inputs.plugin_name }}-plugin-*-SNAPSHOT.jar --destination /var/tmp/bundles
+      - name: Start stack
+        run: |
+          mkdir -p /var/tmp/tomcat_logs
+          chmod ugo+rwx /var/tmp/tomcat_logs
+          cd docker
+          docker-compose -p it -f ${{ matrix.docker-compose-file }} up --no-start
+          docker start it_db_1
+      - name: Wait for MySQL
+        if: ${{ matrix.docker-compose-file == 'docker-compose.ci.mysql.yml' }}
+        run: |
+          set +e
+          count=0
+          until mysqladmin ping -h 127.0.0.1 -u root --password=root --silent; do
+            if [[ "$count" == "25" ]]; then
+              exit 1
+            fi
+            (( count++ ))
+            printf '.'
+            sleep 5
+          done
+          set -e
+      - name: Wait for PostgreSQL
+        if: ${{ matrix.docker-compose-file == 'docker-compose.ci.postgresql.yml' }}
+        run: |
+          set +e
+          count=0
+          until $(psql -h 127.0.0.1 -U postgres -p 5432 -l > /dev/null); do
+            if [[ "$count" == "25" ]]; then
+              exit 1
+            fi
+            (( count++ ))
+            printf '.'
+            sleep 5
+          done
+          set -e
+      - name: Install plugin specific MySQL DDL
+        if: matrix.docker-compose-file == 'docker-compose.ci.mysql.yml' && inputs.ddl_file != ''
+        run: |
+          cat ${{ inputs.ddl_file }} | mysql -h 127.0.0.1 -u root --password=root killbill
+      - name: Install plugin specific PostgreSQL DDL
+        if: matrix.docker-compose-file == 'docker-compose.ci.postgresql.yml' && inputs.ddl_file != ''
+        run: |
+          cat ${{ inputs.ddl_file }} | psql -h 127.0.0.1 -U postgres -p 5432 -d killbill
+      - name: Start Kill Bill
+        # Sometimes it gets stuck (if Kill Bill starts when the DB isn't ready?)
+        timeout-minutes: 4
+        run: |
+          docker start it_killbill_1
+          count=0
+          until $(curl --connect-timeout 10 --max-time 30 --output /dev/null --silent --fail http://${KB_ADDRESS}:${KB_PORT}/1.0/healthcheck); do
+            if [[ "$count" == "180" ]]; then
+              exit 64
+            fi
+            count=$(( count + 1 ))
+            sleep 1
+          done
+          curl --connect-timeout 10 --max-time 30 -v \
+               -X POST \
+               -u admin:password \
+               -H 'Content-Type: application/json' \
+               -H 'X-Killbill-CreatedBy: GitHub' \
+               -d '{"apiKey": "bob", "apiSecret": "lazar"}' \
+               "http://${KB_ADDRESS}:${KB_PORT}/1.0/kb/tenants"
+      - name: Verify plugin has started
+        run: |
+          sudo snap install yq
```

Comment:
Do we use `yq` ?

Reply:
Interesting... maybe not anymore, let me remove it.